packages/azure-services/src/azure-queue/queue.spec.ts

```diff
@@ -235,6 +246,12 @@ describe(Queue, () => {
         });
     });
 
+    function setupQueueGetCount(count: number): void {
+        const getProperties = { approximateMessagesCount: count } as Models.QueueGetPropertiesResponse;
+        queueURLMock.setup(async q => q.getProperties(Aborter.none)).returns(async () => Promise.resolve(getProperties));
+        deadQueueURLMock.setup(async q => q.getProperties(Aborter.none)).returns(async () => Promise.resolve(getProperties));
```

Comment:
why do you have to set the getProperties on deadQueue url t return the same message count?